app/models/requests/mediated_page.rb

```diff
@@ -42,6 +42,8 @@ def item_limit
   end
 
   def requires_needed_date?
+    return false if origin == 'HOPKINS'
+    return false if origin_location == 'PAGE-MP'
```

Comment:
Should we add a configuration for these too? It might make the code a little more clear, if nothing else.